### ISSUE TITLE
'titlehead--draft' missing on drafts for news

### ISSUE THREAD

**Issue by Kariton** [bowman2001/perplex#164]
Hello,

I just got started with Hugo and i am evaluating your template for my personal website.
This template is impressive and very versatile! I think it will fit all of my needs with ease!

While browsing the perplexdoc pages and their mardown files locally i think that the "draft" css property is missing from "news".

i'm comparing:
- http://localhost:1313/doc/security/privacy
- http://localhost:1313/news/hugo-mod-mermaid

the first one has a neat banner indicating that it is a draft, the later does not.

```html
    <div class="titlehead titlehead--draft md md--sans md--tiny">
      <p><strong>DRAFT</strong></p>
    </div></div>
```

**Comment by bowman2001** [comment]
Yes, the partial for a title head which includes the automatic draft head is missing in the template for single news pages. The template needs an overhaul because the layout is buggy.
